tests/functional/host_check/__init__.py

```diff
@@ -0,0 +1,37 @@
+# coding=utf-8
+# -----------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -----------------------------------------------------------------------------
+
+"""Setup for the test package clang_features."""
+
+import os
+import shutil
+
+from libtest import env
+
+# Test workspace should be initialized in this module.
+TEST_WORKSPACE = None
+
+
+def setup_package():
+    """Setup the environment for testing ctu."""
+
+    global TEST_WORKSPACE
+    TEST_WORKSPACE = env.get_workspace('host_check')
```

Comment:
I do not think `TEST_WORKSPACE` setup is needed for these tests. `setup` and `teardown` could just `pass`.

Reply:
Fixed.